components/BottomSheet/README.md

```diff
@@ -99,7 +102,5 @@ button.addTarget(self, action: #selector(presentBottomSheet), for: .touchUpInsid
 ```objc
 _button = [[UIButton alloc] initWithFrame:CGRectZero];
 [_button addTarget:self action:@selector(presentBottomSheet) forControlEvents:UIControlEventTouchUpInside];
-
 ```
-
 <!--</div>-->
```

Comment:
Please make the same changes to `components/BottomSheet/docs/` (the source content for this file).

Reply:
Will do sorry about that!

Reply:
Gentle ping :)